examples/new_model.py

```diff
@@ -42,16 +42,10 @@ def initialize(self, u, z):
             }
     
     def dx(self, x, u):
-        # apply_process_noise is used to add process noise to each step
-        return self.apply_process_noise({
-            'x': x['v'],
-            'v': self.parameters['g'] # Acceleration of gravity
-        })
+        return x
 
     def output(self, x):
-        return self.apply_measurement_noise({
-            'x': x['x']
-        })
+        return x
```

Comment:
Same as above, here you're returning state (x) instead of output (z = {'x': x['x'})